_data/members.yaml

```diff
@@ -177,7 +177,6 @@ chiantipy:
 coin:
   name: COIN
   logo: coin.png
-  url: https://asaip.psu.edu/organizations/iaa/iaa-working-group-of-cosmostatistics/
```

Comment:
@migueldvb would you know a better URL? Related with #239

Reply:
The new URL is https://cosmostatistics-initiative.org/

Reply:
Awesome! Thanks @migueldvb and @emilleishida